module/data/item/fields/item-type-field.mjs

```diff
@@ -1,25 +1,29 @@
 /**
  * A field for storing Item type data.
+ *
+ * @param {object} [defaults={}]                   Options to configure this field's behavior.
+ * @param {string} [defaults.value]                An initial value for the Item's type.
+ * @param {string} [defaults.subtype]              An initial value for the Item's subtype.
+ * @param {string} [defaults.baseItem]             An initial value for the Item's baseItem.
+ * @param {DataFieldOptions} [schemaOptions]       Options forwarded to the SchemaField.
+ * @param {boolean} [schemaOptions.subtype=true]   Include the subtype field.
+ * @param {boolean} [schemaOptions.baseItem=true]  Include the baseItem field.
```

Comment:
Rather than mix in these extra options with the `DataFieldOptions`, could we keep them as part of the first argument, which is more intended to be special options for this field in particular. Perhaps we could change the types to `{string|false}` where explicit `false` indicates the field should be omitted from the schema?